### DISCUSSION TITLE
cross-seed seems to be snatching every torrent it searches for

### DISCUSSION THREAD

**Discussion by OneHelluvaGuy** [cross-seed/cross-seed#844] [Q&A] [answered]
Hello all, I'm new to cross-seed and to the *arrs, so please bear with me. I've got cross-seed set up with Prowlarr and it seems to be snatching every torrent it searches for from my qBit backup directory rather than just matches. I have notifications set up in Pushover and I'm getting constant notifications that a release is grabbed when I start the daemon, even if no new torrents are being injected into the client. At least one tracker profile page is showing the cross-seed searches in the "Recent Snatches" section, all from already active torrents in my client.

Is this the intended behavior? It seems like a heavy load to put on trackers to DL the torrent file of every search. Have I forgotten something in my config? If there's any information I can provide from my config that would be helpful, please let me know; I apologize in advance for my lack of knowledge, or if this is something covered in documentation.

**Comment by zakkarry** [comment]
If you are running the latest, it is impossible that it is snatching every torrent returned in results.

What matchmode are you using? For safe, it will only match matching names of torrents, risky will only snatch things within the fuzzySizeThreshold percentage of different, same applies to partial.

You'd need to show verbose logs to show us what you're seeing.

**Reply by OneHelluvaGuy** [reply]
Thanks for your quick response! I am using matchMode: "safe", and looking at the verbose logs myself, I don't really see anything to indicate that these .torrent files are being downloaded, but again, I'm a noob. I've attached the logs
[verbose.2024-12-20.log](https://github.com/user-attachments/files/18214977/verbose.2024-12-20.log)
in case I'm missing something.

The only indication I'm getting that this is happening is that I'm getting these notifications from my Prowlarr instance that say, "Release Grabbed - [torrent name] grabbed by CrossSeed from [tracker name]". I've gotten 300+ of these notifications since starting the daemon a couple hours ago. It's possible this is a non-issue or an issue with Prowlarr or the notifications there if there's nothing to indicate anything out of the ordinary in my logs, so I'll continue to investigate on that end or disable notifications.

**Reply by ShanaryS** [reply, marked as answer]
Most of your post snatch rejections have been due to it existing in your client. Then the rest are slight differences in file sizes that can only be discovered post snatch. It seems nothing is wrong here. Torrents should only be snatched once and used for the lifetime of cross-seed so the only concern would be duplicate snatches from the same tracker.

**Reply by OneHelluvaGuy** [reply]
Thank you very much for taking a look, that puts my mind at ease. I think there's probably no issue here. Thanks again for taking the time to answer a silly question.
